README.md

```diff
@@ -58,90 +58,57 @@ Write [a config file](#Configuration) directly, or generate it with
 
 Then start the `xremap` daemon by running:
 
-```
+```sh
 sudo xremap config.yml
 ```
 
-(You will need to leave it running for your mappings to take effect.)
+You might need to run it with sudo for application-specific remaps to work in
+certain platforms.
 
-<details>
-<summary>If you want to run xremap without sudo, click here.</summary>
+Make sure to run `xremap` on startup.
 
-### Running xremap without sudo
+### Run without sudo
 
-To do so, your normal user should be able to use `evdev` and `uinput` without sudo.
-In Ubuntu, this can be configured by running the following commands and rebooting your machine.
+To use it without sudo just add your user to the `input` group:
 
-```bash
+```sh
 sudo gpasswd -a YOUR_USER input
-echo 'KERNEL=="uinput", GROUP="input", TAG+="uaccess"' | sudo tee /etc/udev/rules.d/input.rules
 ```
 
-#### Arch Linux
+While this is enough for some distros (Ubuntu), others (Fedora, Arch, Debian)
```

Comment:
> While this is enough for some distros (Ubuntu)

It's not. As indicated by the current README, `echo 'KERNEL=="uinput", GROUP="input", TAG+="uaccess"' | sudo tee /etc/udev/rules.d/input.rules` is necessary, at least for Ubuntu.